test/suites/integration/bsp/submit-proofs.test.ts

```diff
@@ -195,48 +195,45 @@ describeBspNet(
       );
     });
 
-    it(
-      "BSP can correctly delete a file from its forest and runtime correctly updates its root",
-      { skip: "Not implemented yet. Needs RPC method to build proofs." },
-      async () => {
-        // TODO: Setup a BSP that has two files which lie under the same NibbledBranch in the forest.
-        // TODO: Generate the proof to delete one of the files.
-        /* let inclusionForestProof = bspThreeApi.rpc.storagehubclient.buildForestRoot(fileMetadata.fileKey); */
-        // TODO: Request the deletion of the file:
-        /* const fileDeletionRequestResult = bspThreeApi.sealBlock(bspThreeApi.tx.fileSystem.bspRequestStopStoring(
-            fileMetadata.fileKey,
-            fileMetadata.bucketId,
-            fileMetadata.location,
-            fileMetadata.owner,
-            fileMetadata.fingerprint,
-            fileMetadata.fileSize,
-            false,
-            inclusion_forest_proof: ForestProof<T>,
-        ); */
-        // Wait enough blocks for the deletion to be allowed.
-        /* const currentBlock = await bspThreeApi.rpc.chain.getBlock();
-    const currentBlockNumber = currentBlock.block.header.number.toNumber();
-    const cooldown = currentBlockNumber + bspThreeApi.consts.fileSystem.minWaitForStopStoring.toNumber();
-    await bspThreeApi.block.skipTo(cooldown); */
-        // TODO: Confirm the request of deletion. Make sure the extrinsic doesn't fail and the root is updated correctly.
-        /*  const fileDeletionConfirmResult = bspThreeApi.sealBlock(bspThreeApi.tx.fileSystem.bspConfirmStopStoring(
-        fileMetadata.fileKey,
-        inclusionForestProof,
-      ));
+    it("BSP can correctly delete a file from its forest and runtime correctly updates its root", async () => {
+      // TODO: Setup a BSP that has two files which lie under the same NibbledBranch in the forest.
```

Comment:
This comment was probably leftover from something else we wanted to test. We should probably remove it or put it somewhere else so that it doesn't confuse future readers.